panoramapublic/test/src/org/labkey/test/tests/panoramapublic/PanoramaPublicTest.java

```diff
@@ -291,12 +291,12 @@ public DataRegionTable getDataRegion()
 
         public void submitWithoutPXId()
         {
-            findElement(Locator.linkContainingText("Submit")).click();
-            waitAndClick(Locator.linkContainingText("Continue without a ProteomeXchange ID"));
+            clickAndWait(Locator.linkContainingText("Submit"));
+            clickAndWait(Locator.linkContainingText("Continue without a ProteomeXchange ID"));
             getWrapper()._ext4Helper.selectComboBoxItem(Ext4Helper.Locators.formItemWithInputNamed("journalId"), PANORAMA_PUBLIC);
-            waitAndClick(Ext4Helper.Locators.ext4Button("Submit"));
-            waitAndClick(Locator.lkButton("OK")); // Confirm to proceed with the submission.
-            waitAndClick(Locator.linkWithText("Back to Experiment Details")); // Navigate to the experiment details page.
+            clickAndWait(Ext4Helper.Locators.ext4Button("Submit"));
```

Comment:
I'm guessing these don't need to be a `waitAndClickAndWait` because the action before this was a `clickAndWait`. Correct?